scripts/versions-to-update.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+set -eu
+
+usage() {
+  cat << USAGE >&2
+Usage:
+  $0 <package> <version>
+USAGE
+}
+
+if [[ $# -lt 2 ]]; then
+  usage
+  exit 1
+fi
+
+get_versions() {
+  local package="$1"
+  local version="$2"
+  local major=$(semver get major "$version")
+  local minor=$(semver get minor "$version")
+  local prerel=$(semver get prerel "$version")
+  local latestVersion=$(npm view $package version)
```

Comment:
Nitpick but the mix of camel/snake case is unusual in bash scripts.
Also might be worth running shellcheck on this.

Reply:
Ah yes definitely can fix that, I rarely write bash scripts, I don't know the best practices. I'll look into it and make the necessary changes.